client/Dockerfile

```
@@ -19,5 +19,9 @@ ARG SENTRYCLI_USE_LOCAL=1
 RUN npm install --location=global --unsafe-perm
 COPY . .
 
-EXPOSE 8080
-CMD ["gulp", "watch"]
\ No newline at end of file
+RUN npm install
+# prebuild, for CI optimization
+RUN gulp public javascript worker wasm
+
+CMD ["gulp", "start"]
```

The default cmd should be `gulp watch` for development. `start` doesn't update when you change the files.

```suggestion
CMD ["gulp", "watch"]
```

If you want to change it to `start` for CI only then you can create an [override](https://docs.docker.com/compose/extends/#multiple-compose-files) for `docker-compose.yml` or run `docker-compose run client gulp start`

---

Changed, seems pass all test, but few time integration tests timeouted. Lets see how it will run long run.